common/src/main/scala/uk/ac/wellcome/models/UnifiedItem.scala

```diff
@@ -11,10 +11,20 @@ case class SourceIdentifier(source: String, sourceId: String, value: String)
 
 case class IdentifiedUnifiedItem(canonicalId: String, unifiedItem: UnifiedItem)
 
-/** A representation of an item in our ontology, without a canonical identifier */
-case class UnifiedItem(identifiers: List[SourceIdentifier],
-                       label: String,
-                       accessStatus: Option[String] = None)
+/** A representation of an item in our ontology, without a
+ *  canonical identifier.
+ */
+case class UnifiedItem(
+  identifiers: List[SourceIdentifier],
+  label: String,
+  description: Option[String] = None,
+  lettering: Option[String] = None,
+  hasCreatedDate: Option[String] = None,
```

Comment:
I'm torn between empty and using a simple string representation so we can get something out the other side. We are going to have to reindex a lot any way.